modules/gdscript/gdscript_vm.cpp

```diff
@@ -2649,6 +2649,8 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 				GET_VARIANT_PTR(counter, 0);
 				GET_VARIANT_PTR(container, 1);
 
+				*counter = Variant(); // TODO: Fix in compiler/codegen?
```

Comment:
Please remove the `TODO` comment. I think it's fine doing it here, as doing with the codegen would have to execute yet another instruction.